kitsune/customercare/zendesk.py

```diff
@@ -42,6 +50,11 @@ def _user_to_zendesk_user(self, user, email):
             external_id=external_id,
         )
 
+    def get_user_by_email(self, email):
+        """Given an email, return a user from Zendesk."""
+        for user in self.client.search(type="user", query=f"email:{email}"):
```

Comment:
What happens when we get a response with multiple users?

Reply:
It shouldn't be possible, as Zendesk uses the email as a primary id for users.
https://support.zendesk.com/hc/en-us/articles/4408831446298-Can-user-accounts-share-an-email-address-